iothub/device/src/Transport/AmqpIoT/AmqpIoTConstants.cs

```diff
@@ -25,14 +25,14 @@ internal static class AmqpIoTConstants
         internal static readonly Accepted AcceptedOutcome = AmqpConstants.AcceptedOutcome;
 
         internal const string ResponseStatusName = "status";
-        internal const string TelemetrySenderLinkSuffix = "_TelemetrySenderLink";
-        internal const string TelemetryReceiveLinkSuffix = "_TelemetryReceiverLink";
-        internal const string EventsReceiverLinkSuffix = "_EventsReceiverLink";
-        internal const string MethodsSenderLinkSuffix = "_MethodsSenderLink";
-        internal const string MethodsReceiverLinkSuffix = "_MethodsReceiverLink";
+        internal const string TelemetrySenderLinkSuffix = "TelemetrySenderLink";
```

Comment:
What are these used for? 
I see they are passed in `AmqpLinkSettings` but is the format relevant? Do the `linkName`s need to be unique?
If they are sent over the wire, can we make them a lot shorter?

Reply:
The link name is unique per session. I think it is mapped to a short while been sent to wire.